tools/Utilities/src/ViewModels/UtilityViewModel.cs

```diff
@@ -28,7 +31,20 @@ public class UtilityViewModel : INotifyPropertyChanged
 
     public ICommand LaunchCommand { get; set; }
 
-    public ICommand LaunchAsAdminCommand { get; set; }
+    public Visibility LaunchAsAdminVisibility { get; set; }

```

Comment:
This can be a bool.  I believe WinUI 3 can convert bools to visibility implicitly.

Reply:
consider using "SupportsLaunchAsAdmin" or something instead of "LaunchAsAdminVisibility" that describes what this means rather than what the view is going to do in response.

Reply:
> This can be a bool. I believe WinUI 3 can convert bools to visibility implicitly.

As long as you use x:bind